src/applications/apps.js

```diff
@@ -443,6 +443,11 @@ export function toDynamicPathValidatorRegex(path) {
     regexStr += inDynamic
       ? anyCharMaybeTrailingSlashRegex
       : commonStringSubPath;
+
+    if (index === path.length && !inDynamic) {
+      regexStr = `(${regexStr}$)|(${regexStr}\\/.*$)`;
```

Comment:
Could this be simplified to the following?

```suggestion
      regexStr += '(\/.*)?$';
```

Reply:
It could not match `/pathname/` who already has a slash.
ie `/pathname/` to match `/pathname/extra`